File: Java_etc/jdbc_correction/src/main/java/model/Article.java
package model;

public abstract class Article {
	private Integer numeroArticle;
	private Integer nbDisques;
	private Adherent emprunteur;

	public Article() {
	}

	/**
	 * @param numeroArticle
	 * @param nbDisques
	 */
	public Article(Integer numeroArticle, Integer nbDisques) {
		this.numeroArticle = numeroArticle;
		this.nbDisques = nbDisques;
	}

	/**
	 * @param nbDisques
	 */
	public Article(Integer nbDisques) {
		this.nbDisques = nbDisques;
	}

	public Integer getNumeroArticle() {
		return numeroArticle;
	}

	public void setNumeroArticle(Integer numeroArticle) {
		this.numeroArticle = numeroArticle;
	}

	public Integer getNbDisques() {
		return nbDisques;
	}

	public void setNbDisques(Integer nbDisques) {
		this.nbDisques = nbDisques;
	}

	public Adherent getEmprunteur() {
		return emprunteur;
	}

	public void setEmprunteur(Adherent emprunteur) {
		this.emprunteur = emprunteur;
	}
	
	

}

File: angularCorrectionThibault/src/app/model/article.ts
export abstract class Article {

  constructor(protected _id?: number, protected _nbDisques?: number) {

  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }


  get nbDisques(): number {
    return this._nbDisques;
  }

  set nbDisques(value: number) {
    this._nbDisques = value;
  }
}
